Author: Niko Tyni <ntyni@iki.fi>
Description: Increase the time for the backend to start so slower buildds
have a chance. (#418717)
--- a/speedy/t/killfe.t
+++ b/speedy/t/killfe.t
@@ -9,7 +9,7 @@
 my $cmd = "exec $ENV{SPEEDY} -- -M1 $scr </dev/null |";
 
 my $pid = open (RUN1, $cmd);
-sleep(1);
+sleep(3);
 kill(9, $pid);
 wait;
 open (RUN2, $cmd);
